docs/src/modules/operations/pages/stanalone.adoc

```diff
@@ -0,0 +1,183 @@
+= Running Akka without the full platform
+
+include::ROOT:partial$include.adoc[]
+
+ifdef::todo[TODO: intro]
+
+== License key
+
+You can retrieve a free license key from akka.io[https://account.akka.io/key]. Select the Akka SDK option.
+
+Define the key in your `application.conf`:
+
+```
+akka.license-key = "<your key>"
+```
+
+== Database
+
+Akka requires a Postgres database. The tables will be created automatically but you need to create the database and the credentials to connect to the database.
+
+You define connection configuration in your `application.conf` or use the predefined environment variables.
+
+```
+akka.persistence.r2dbc {
+  connection-factory {
+    host = "localhost"
+    host = ${?DB_HOST}
+    port = 5432
+    port = ${?DB_PORT}
+    database = "postgres"
+    database = ${?DB_DATABASE}
+    user = "postgres"
+    user = ${?DB_USER}
+    password = ${?DB_PASSWORD}
+  }
+}
+```
+
+Supported environment variables:
+
+- DB_HOST
+- DB_PORT
+- DB_DATABASE
+- DB_USER
+- DB_SSL_ENABLED - `true` to enable SSL
+- DB_SSL_MODE - allow, prefer, require, verify-ca, verify-full, tunnel
+- DB_SSL_ROOT_CERT - Server root certificate. Can point to either a resource within the classpath or a file.
+- DB_SSL_CERT - Client certificate. Can point to either a resource within the classpath or a file.
+- DB_SSL_KEY - Key for client certificate. Can point to either a resource within the classpath or a file.
+- DB_SSL_PASSWORD - Password for client key.
+
+You find more configuration options in https://doc.akka.io/libraries/akka-persistence-r2dbc/current/config.html[reference documentation].
+
+== Build a container image
+
+Build a container image of the service for standalone use by activating the `standalone` maven profile:
+
+[source, command line]
+----
+mvn clean install -DskipTests -Pstandalone
+----
+
+Push the image to your container registry.
+
+== Deploy to Kubernetes
```

Comment:
Could it be worth a showing a docker compose example file as well, for easy testing outside of k8?

Reply:
I had one, but thought it could be misleading to show that as a local dev alternative, where it's much better to just use `mvn exec:java`.